src/llmq/quorums_chainlocks.h

```diff
@@ -24,7 +24,7 @@ class CChainLockSig
 {
 public:
     int32_t nHeight{-1};
-    uint256 blockHash;
+    uint256 blockHash{uint256()};
```

Comment:
no need for default initializer as the uint256 constructor already does zero initialization